utils/client.js

```diff
@@ -2,13 +2,12 @@ import ApolloClient from 'apollo-boost';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 
 const PORT = process.env.PORT || 3000;
-const path = process.env.ENV;
-const basePath =
-  path === 'development'
-    ? `http://localhost:${PORT}`
-    : 'https://vcars.herokuapp.com';
-
-    export const client = new ApolloClient({
-  uri: `${basePath}/api/graphql`,
+const path =
```

Comment:
Codacy has a fix for the issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7132271)
```suggestion
const path =  process.env.NODE_ENV === 'production'
```